src/screens/BB-BUS/DetailScreen.js

```diff
@@ -57,326 +40,214 @@ const TEXT_BLOCKS_SORTER = {
-              )}
-
-              {!!firstTextBlocks &&
-                firstTextBlocks.map((textBlock, index) =>
-                  renderTextBlock(firstTextBlocks, textBlock, index)
-                )}
-
-              {!!authorities &&
-                authorities.map((authority, index) => (
-                  <Authority
-                    key={authority.authority.id}
-                    data={authority.authority}
-                    accordion={accordion}
-                    onPress={() => {
-                      // set the accordion part active/visible or not for the clicked element
-                      // identified by authority id
-                      accordion[authority.authority.id] = !accordion[authority.authority.id];
-                      setAccordion({ ...accordion });
-                    }}
-                    bottomDivider={
-                      !!accordion[authority.authority.id] || index == authorities.length - 1
-                    }
-                    openWebScreen={openWebScreen}
-                  />
-                ))}
-
-              {!!persons && !!persons.length && (
-                <Persons
-                  data={{ id: details.id, persons }}
-                  accordion={accordion}
-                  onPress={() => {
-                    // set the accordion part active/visible or not for the clicked element
-                    // identified by service id
-                    accordion[details.id] = !accordion[details.id];
-                    setAccordion({ ...accordion });
-                  }}
-                  openWebScreen={openWebScreen}
-                  orientation={orientation}
-                  dimensions={dimensions}
-                />
-              )}
-
-              {!!sortedTextBlocks &&
-                sortedTextBlocks.map((textBlock, index) =>
-                  renderTextBlock(sortedTextBlocks, textBlock, index)
-                )}
-
-              {showBackToTop && (
-                <BackToTop
-                  onPress={() =>
-                    scrollViewRef.current.scrollTo({
-                      x: 0,
-                      y: 0,
-                      animated: true
-                    })
-                  }
-                />
-              )}
-            </ScrollView>
-          </SafeAreaViewFlex>
-        );
-      }}
-    </Query>
+          />
+        )}
+      </ScrollView>
+    </SafeAreaViewFlex>
   );
-  /* eslint-enable complexity */
 };
 
 const styles = StyleSheet.create({
-  sectionTitle: {
-    backgroundColor: colors.transparent,
-    paddingHorizontal: normalize(14),
-    paddingVertical: normalize(12)
-  },
   formContainer: {
     marginTop: normalize(21),
     paddingHorizontal: normalize(14)
   }
 });
 
 DetailScreen.propTypes = {
-  navigation: PropTypes.object.isRequired,
   route: PropTypes.object.isRequired
 };
+
+FormButton.propTypes = {
+  link: PropTypes.object.isRequired,
+  rootRouteName: PropTypes.string.isRequired
+};
```

Comment:
i would put the prop types beneath its component, so this could be places on top of the file next to `FormButton`

Reply:
done